django/utils/cache.py

```diff
@@ -382,8 +383,8 @@ def get_cache_key(request, key_prefix=None, method="GET", cache=None):
     account from the global URL registry and uses those to build a cache key
     to check against.
 
-    If there isn't a headerlist stored, return None, indicating that the page
-    needs to be rebuilt.
+    If there is no headerlist stored, the page needs to be rebuilt, so this
+    function returns ``None``.
```

Comment:
This seems unrelated. Could either be dropped or merged as a separate clean up. (Pending Fellow decisions)